src/main/java/de/hysky/skyblocker/config/configs/MiscConfig.java

```diff
@@ -0,0 +1,35 @@
+package de.hysky.skyblocker.config.configs;
+
+import dev.isxander.yacl3.config.v2.api.SerialEntry;
+import net.minecraft.client.resource.language.I18n;
+
+public class MiscConfig {
+    @SerialEntry
+    public boolean hideStatusEffectOverlay = false;
```

Comment:
Given that this hides vanilla potion effects in the inventory screen it probably belongs in UI